storage/localstore/mode_put.go

```diff
@@ -108,7 +108,26 @@ func (db *DB) put(mode chunk.ModePut, chs ...chunk.Chunk) (exist []bool, err err
 			}
 			gcSizeChange += c
 		}
-
+	case chunk.ModePutReUpload:
+		// TODO: this should do the same as `ModePutUpload`, but also add the chunk hash to the pushindex
```

Comment:
put it to pushindex only no?

Reply:
removed this in favour of `mode.ModeSetReUpload`